test/test_dsa_store.py

```diff
@@ -33,6 +34,98 @@ def test_put_get(self):
         rtn_dict = my_etcd.get_dict('/test/1')
         self.assertEqual(rtn_dict, test_dict)
 
+    def put_bad_val(self, val):
```

Comment:
general comment. If these test keys are being placed in the production etcd service, then they should be deleted when the test is complete.